core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -19,6 +19,13 @@ package kafka.controller
 import java.util.concurrent.atomic.AtomicInteger
 import java.util.concurrent.{LinkedBlockingQueue, TimeUnit}
 
+import org.apache.kafka.common.errors.{BrokerNotAvailableException, ControllerMovedException}
+import org.apache.kafka.common.protocol.{ApiKeys, Errors}
+import org.apache.kafka.common.TopicPartition
+import org.apache.kafka.common.requests.{AbstractRequest, AbstractResponse, LeaderAndIsrResponse}
+
+import scala.collection.JavaConverters._
```

Comment:
This rule doesn't apply to `JavaConverters` since you need explicit `asScala` and `asJava` calls anyway. That rule was for `JavaConversions` which did things automatically and has been deprecated.